CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,57 @@
+
+- **Harassment, public or private, is unacceptable and won’t be tolerated.** This includes but is not limited to:
+  - Intimidation
+  - Stalking
+  - Unwelcome following
+  - Enlisting the help of others, whether in person or online, in order to target an attendee
+  - Taking photographs, video, or audio recordings or recordings without consent
+  - Obscene or intimidating gestures
+  - Shouting
+  - Sustained disruption of talks and events
+  - Disruption of meetings
+  - Inappropriate physical contact
+  - Unwelcome sexual attention
+  - Posting (or threatening to post) other people's personally identifying information ("doxing")
+
+
+- No matter who you are, if you feel you have been or are being harassed or made uncomfortable by another person in relation to your work on FPF projects, please report it immediately.  We are aware that harassment and sexual violence are systemic and pervasive social issues that demand continued awareness and attention, and that a code of conduct is only a small component of that.  We are committed to taking proactive action to make our organizational spaces safe and comfortable for everyone.
+
+
+- **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Exclusionary behavior is absolutely unacceptable. This includes, but is not limited to:
+  - Discriminatory jokes and language.
+  - Posting sexually explicit or violent material.
+  - Personal insults, especially those using racist or sexist terms.
+  - Unwelcome sexual attention.
+  - Advocating for, or encouraging, any of the above behavior.
+
+
+- **When we disagree, try to understand why.** Disagreements — both social and technical — happen all the time, even at Freedom of the Press Foundation. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of FPF comes from its varied team, people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn't mean that they’re wrong. Don’t forget that it is human to err and blaming each other doesn't get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
```

Comment:
Nit, we should not use stylized single or double quotes. They should be `0x2C` and `0x22` respectively.

> that we’re different

This might just be a rendering issue on GH, however.